sync-core/src/main/java/com/cloudant/sync/replication/PullReplication.java

```diff
@@ -15,7 +15,7 @@
 public class PullReplication extends Replication {
 
     /**
-     * URI for this replication's remote database.
+     * URI for this replication's remote database (including username and password if necessary).
```

Comment:
Could this say "Include username and password in the URL, or supply an Authorization header using optionalHeaders"?